docs/security-considerations.rst

```diff
@@ -181,13 +181,14 @@ Sending and Receiving Ether
      contract. Again, the best practice here is to use a :ref:`"withdraw"
      pattern instead of a "send" pattern <withdrawal_pattern>`.
 
-Callstack Depth
-===============
+Call stack Depth
+================
 
 External function calls can fail any time because they exceed the maximum
 call stack of 1024. In such situations, Solidity throws an exception.
 Malicious actors might be able to force the call stack to a high value
-before they interact with your contract.
+before they interact with your contract, by controlling the number of calls
+and local function variables on the stack.
```

Comment:
Maybe we should say that 63/64 rule makes call stack depth attacks impractical. What do you think @chriseth?

Reply:
Yes, but should mention as "Starting the Spurious Dragon hard fork, the 63/64th rule ..." or whichever HF it was. Also link to the EIP.

Reply:
We should also clarify that the call stack and the expression stack are two totally unrelated things that are both limited to 1024. I guess this confusion started the PR.